charts/velero/templates/backupstoragelocation.yaml

```diff
@@ -4,7 +4,7 @@ kind: BackupStorageLocation
 metadata:
   name: {{ include "velero.backupStorageLocation.name" . }}
   annotations:
-    "helm.sh/hook": post-install,post-upgrade
+    "helm.sh/hook": post-install,post-upgrade,post-rollback
```

Comment:
Are the `helm.sh/hook` changes here and in other files relevant to this change? I'm not super familiar with helm lifecycle management so I don't know if there's something I'm missing here :)

Reply:
Since helm support `upgrade` and `rollback`.
I thought that the user is possible to roll back to the previous version once the user found any issue in the newer version.
Therefore, the CRDS should be rollback to the previous one also.

Reply:
Thanks, that makes sense!